experiment/run_experiment.py

```diff
@@ -524,6 +593,10 @@ def main():
                         '--runners-cpus',
                         help='Cpus available to the runners.',
                         required=False)
+    parser.add_argument('-rs',
+                        '--random-seed-corpus',
+                        help='Path to the random seed corpus',
```

Comment:
Is this a path to the zip file with all the seed corpora?

Reply:
Yes, the full command for this experiment looks like this:
```
PYTHONPATH=. python3 experiment/run_experiment.py \
--experiment-config /local/pfm/local-experiment-config.yaml \
--benchmarks sqlite3_ossfuzz  libjpeg-turbo-07-2017 \
--experiment-name my-sample-expriment \
--fuzzers afl entropic \
--random-seed-corpus /local/exp/my_corpus
```

where inside the `/local/exp/my_corpus` directory there must be sub-directory containing seed corpus for all running benchmarks:
```
-- my_corpus
---- sqlite3_ossfuzz  
-------- fbeb1de0be72d83ed8dff924fd9df8cedb0a7497
-------- fba1b82f6612e8e1d1786b757dfeecbbfabeeca5
---- libjpeg-turbo-07-2017
-------- f78602c270b787d3e7e940f50f78365552d5a20b
-------- ebc89a181d637f6bdfdcaa9d647cfd1f88d62675
-------- ed0fbdb5c9f7d2fbb1c6c8e0665e80b110ba1e14
```

Reply:
Yes, each zipped benchmark corpus is kept in a [newly generated directory ](https://github.com/Practical-Formal-Methods/fuzzbench-public/pull/2/files#diff-f29fa4fe4e55f233de1357ff219fd6df4fc408fcbdd0406f3fb5a022e11c670bR69) named `random_seed_corpora_zip`.
```
|— my_corpus
|    |— sqlite3_ossfuzz  
|    |    |— fbeb1de0be72d83ed8dff924fd9df8cedb0a7497
|    |    |— fba1b82f6612e8e1d1786b757dfeecbbfabeeca5
|    |— libjpeg-turbo-07-2017
|    |    |— f78602c270b787d3e7e940f50f78365552d5a20b
|    |    |— ebc89a181d637f6bdfdcaa9d647cfd1f88d62675
|    |    |— ed0fbdb5c9f7d2fbb1c6c8e0665e80b110ba1e14
|    |— random_seed_corpora_zip
|    |    |— sqlite3_ossfuzz.zip
|    |    |    |— fbeb1de0be72d83ed8dff924fd9df8cedb0a7497
|    |    |    |— fba1b82f6612e8e1d1786b757dfeecbbfabeeca5
|    |    |— libjpeg-turbo-07-2017.zip
|    |    |    |— f78602c270b787d3e7e940f50f78365552d5a20b
|    |    |    |— ebc89a181d637f6bdfdcaa9d647cfd1f88d62675
|    |    |    |— ed0fbdb5c9f7d2fbb1c6c8e0665e80b110ba1e14
```

Reply:
Thanks for clarifying! Could the zips also be located in `my_corpus` instead of `my_corpus/random_seed_corpora_zip`?

Reply:
fixed